definitions/divorce/json/CaseEvent/CaseEvent.json

```diff
@@ -2302,7 +2315,7 @@
     "PreConditionState(s)": "AwaitingService",
     "PostConditionState": "AosAwaiting",
     "RetriesTimeoutURLAboutToSubmitEvent": "120,120",
-    "CallBackURLSubmittedEvent": "${CCD_DEF_COS_URL}/bulk-print",
+    "CallBackURLSubmittedEvent": "${CCD_DEF_COS_URL}/solicitor-confirm-service",
```

Comment:
are you sure this is correct?

Reply:
https://github.com/hmcts/div-case-orchestration-service/pull/1320/files#diff-a34f8f77ae27cf30ca8266a79417be90d972cf08b2d6339465816c9e790e7a89R70